pkg/logging/cri_logger.go

```diff
@@ -113,56 +113,62 @@ func ReadLogs(ctx context.Context, opts *LogViewOptions, stdout, stderr io.Write
 	writer := newLogWriter(stdout, stderr, opts)
 	msg := &logMessage{}
 	for {
-		if stop || (limitedMode && limitedNum == 0) {
-			logrus.Debugf("Finished parsing log file, path; %s", logPath)
+		select {
+		case <-stopChannel:
+			logrus.Debugf("received stop signal while reading cri logfile, returning")
 			return nil
-		}
-		l, err := r.ReadBytes(eol[0])
-		if err != nil {
-			if err != io.EOF { // This is an real error
-				return fmt.Errorf("failed to read log file %q: %v", logPath, err)
+		default:
+			if stop || (limitedMode && limitedNum == 0) {
+				logrus.Debugf("Finished parsing log file, path; %s", logPath)
+				return nil
```

Comment:
nit: `Finished` -> `finished` to be consistent; and typo `;` -> `:`.

Reply:
According to common custom，maybe it's a good idea to let log print line ( such as `logrus.Debugf(“Example”)` `logrus.Warnf(“Example”)`) start with uppercase, and error string returned (such as`fmt.Errorf("example")`) start with lowercase

And It seems that there is no uniformity about `uppercase` and `lowercase`
`uppercase` : https://github.com/containerd/nerdctl/blob/main/pkg/composer/create.go#L169

`lowercase`: https://github.com/containerd/nerdctl/blob/main/cmd/nerdctl/container_cp_linux.go#L259

Reply:
For reference only: https://www.postgres-xl.org/documentation/error-style-guide.html

I think it is also necessary to unify it in our project.

Reply:
+1  Agree, maybe we should open a issue to track the uniformity in our registry，cc@djdongjin

Reply:
This SGTM, but personally I feel this uniformity is more a best-effort but difficult to achieve (or maintain after achieved), part because it's quite detailed and there is a lot of error/log messages, and also because there are new contributors (and reviewers) who're not aware of these.

Speaking of this PR, I'd suggest to lowercase all error/log messages :) since it has both uppercase (line 122, 146, 152, ...) and lowercase (118, 128, 135, ...).

Reply:
ok, let me lowercase error/log message in this PR